execution/evm/vm.go

```diff
@@ -48,6 +48,7 @@ var (
 	ErrDataStackUnderflow     = errors.New("Data stack underflow")
 	ErrInvalidContract        = errors.New("Invalid contract")
 	ErrNativeContractCodeCopy = errors.New("Tried to copy native contract code")
+	ErrInvalidOpcode          = errors.New("Invalid opcode")
```

Comment:
This has the potential to read as 'an opcode we don't know about', when it means 'abort the execution'. I'd be tempted to suggest: `ErrExecutionAborted = errors.New("Execution aborted")`.